app/code/Magento/Sales/Test/Mftf/Test/StorefrontCustomerOrderDetailsPagerTest.xml

```diff
@@ -0,0 +1,286 @@
+        </actionGroup>
+        <actionGroup ref="StorefrontAddBundleProductToTheCartActionGroup" stepKey="addBundleProductToCart">
+            <argument name="productName" value="$simpleProduct2.name$"/>
+            <argument name="quantity" value="1"/>
+        </actionGroup>
+        <!-- Place the order -->
+        <actionGroup ref="StorefrontOpenCheckoutPageActionGroup" stepKey="onCheckout"/>
+        <see userInput="22" selector="{{CheckoutOrderSummarySection.itemsQtyInCart}}" stepKey="see21Products"/>
+        <click selector="{{CheckoutShippingSection.firstShippingMethod}}" stepKey="selectFirstShippingMethod"/>
+        <actionGroup ref="StorefrontCheckoutClickNextOnShippingStepActionGroup" stepKey="clickNextButton"/>
+        <click selector="{{CheckoutPaymentSection.placeOrder}}" stepKey="placeOrder"/>
+        <waitForPageLoad stepKey="waitForSuccess"/>
+        <grabTextFrom selector="{{CheckoutSuccessMainSection.orderNumber22}}" stepKey="grabOrderNumber"/>
+        <!-- Go to My Account > My Orders page -->
+        <actionGroup ref="StorefrontOpenMyAccountPageActionGroup" stepKey="onMyAccount"/>
+        <click selector="{{StorefrontCustomerSidebarSection.sidebarTab('My Orders')}}" stepKey="clickOnMyOrders"/>
+        <waitForPageLoad stepKey="waitForOrdersLoad"/>
+        <!-- Click 'View Order' link on order from preconditions -->
+        <actionGroup ref="StorefrontCustomerOrdersViewOrderActionGroup" stepKey="clickOrderView">
+            <argument name="orderNumber" value="{$grabOrderNumber}"/>
+        </actionGroup>
+        <!-- Assert that Pager displays 20 order items correctly -->
+        <grabTextFrom selector="{{StorefrontCustomerOrderViewSection.pager}}" stepKey="productsCount"/>
+        <assertStringContainsString stepKey="assertProductsCount">
+            <actualResult type="const">$productsCount</actualResult>
+            <expectedResult type="string">Items 1 to 20 of 22 total</expectedResult>
+        </assertStringContainsString>
+        <!-- Assert that Pager displays 2nd page order items correctly -->
+        <click selector="a.action.next" stepKey="clickNextPage"/>
+        <grabTextFrom selector="{{StorefrontCustomerOrderViewSection.productName}}" stepKey="configOptionValue"/>
+        <assertStringContainsString stepKey="assertConfigurableChildProduct">
+            <actualResult type="variable">configOptionValue</actualResult>
+            <expectedResult type="string">$$getConfigAttributeSecondOption.label$$</expectedResult>
+        </assertStringContainsString>
+        <grabMultiple selector="{{StorefrontCustomerOrderViewSection.productName}}" stepKey="bundleOptionValue"/>
+        <assertContains stepKey="assertBundleChildProduct">
+            <actualResult type="variable">bundleOptionValue</actualResult>
+            <expectedResult type="string">1 x $$simpleProduct2.name$$ $200.00</expectedResult>
+        </assertContains>
```

Comment:
Instead of asserting that configurable and bundle products are on the second page, it may be better to assert that there are 2 items on the second page. It will be simpler and more related to the PR change.

Reply:
Replaced with asserting of two items.